x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/TimeseriesLifecycleType.java

```diff
@@ -413,6 +411,22 @@ public static String validateMonotonicallyIncreasingPhaseTimings(Collection<Phas
         return Strings.collectionToCommaDelimitedString(errors);
     }
 
+    /**
+     * Require that all "frozen" phases configured in a policy have a searchable snapshot action.
```

Comment:
uber nit :) but this implies the possible presence of multiple frozen phases in one policy and it might be confusing

Reply:
Good point, I changed this to not imply that.